Task6/XMLRepository.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System.Xml.Serialization;
+
+namespace Task6
+{
+    public class XMLRepository : IRepository
+    {
+        public void Save(Catalog catalog, string filePath)
+        {
+            XmlSerializer serializer = new XmlSerializer(typeof(CatalogData));
+            using (StreamWriter writer = new StreamWriter(filePath))
+            {
+                serializer.Serialize(writer, CatalogData.FromCatalog(catalog));
+            }
+        }
+
+        public Catalog Load(string filePath)
+        {
+            XmlSerializer serializer = new XmlSerializer(typeof(CatalogData));
+            using (StreamReader reader = new StreamReader(filePath))
+            {
+                CatalogData catalogData = (CatalogData)serializer.Deserialize(reader);
+                return catalogData.ToCatalog();
+            }
+        }
+    }
+
```

Comment:
Good way! This is DAL classes. Add class for authors, move DAL classes to separate files, and use them both for JSON and XML repos